misk-hibernate/src/main/kotlin/misk/hibernate/Transacter.kt

```diff
@@ -49,6 +49,8 @@ interface Transacter {
    *   replicas.
    * * If no reader is configured for replica reads when installing the [HibernateModule], this
    *   method will throw an [IllegalStateException].
+   * * There's another instance of [Transacter] with the reader qualifier that uses the read
```

Comment:
`Note: You can do it another way, where you annotate the [Transacter] with the readerQualifer defined by [SkimHibernateModule], which will use the read only replica as the datasource.`

Reply:
updated